packages/scripts/config/webpack.config.js

```diff
@@ -89,20 +83,38 @@ const config = {
 	},
 	module: {
 		rules: [
-			{
+			// Sourcemap Loader is a complement to the `devtool` configuration,
```

Comment:
Can it be moved next to `devtool` in the source code to keep them glued together?

Reply:
Moved next to `devtool` with https://github.com/WordPress/gutenberg/pull/14860/commits/ba48f76984fba8d895ba16ab47c87ac97e072fca